server.go

```diff
@@ -29,6 +29,9 @@ func StartServer(user, pass, jiraHost string, fc FieldConfig) error {
 	}
 
 	r := gin.Default()
+	if err := r.SetTrustedProxies(nil); err != nil {
+		log.Println("failed to set trusted proxies on gin server")
```

Comment:
Hmm... how serious is this error? Wondering if it should be returned to caller, which would abort startup.

Reply:
Good call - I had done the lazy thing here because `nil` will never panic, but returning the error is more sensible for a future where this could be configurable. It should abort startup for sure since it would mean an invalid address ([source](https://github.com/gin-gonic/gin/blob/d07db174acf44bfaf191ca2f6d7beafa2ff946da/gin.go#L386-L426))

Reply:
https://github.com/andrei-m/jira-graph/pull/28/commits/9fe00bdab2bf24fb375cbf65f1eae1f541df43c6